apps/mobile/src/locales/index.ts

```diff
@@ -41,6 +45,12 @@ export async function initiateI18n() {
   // run load and activate so the I18nProvider doesn't block the render
   i18n.load('en', {});
   i18n.activate(DEFAULT_LOCALE);
+  const locales = getLocales();
+
+  const deviceLocale =
+    locales.find(locale => isDefined(locale.languageTag))?.languageTag ?? DEFAULT_LOCALE;
+  // LEA-2138 Only support es-ES and en
```

Comment:
@pete-watters would it be easy enough to remove this conditional check for `es-ES` so that any local device language loads upon toggle?

Reply:
Tracking [here](https://linear.app/leather-io/issue/LEA-2307/add-language-selector-view-for-setting-languages)

Reply:
Yes we can do that easily enough when the time is right. There's no point right now with only `ES` ready